apps/root/src/common/components/wallet-selector/useWalletSelectorState.tsx

```diff
@@ -407,18 +455,26 @@ const useWalletSelectorState = ({ options, showWalletCounter }: WalletSelectorBa
 
   const selectedOptionLabel =
     selectedOptionValue === ALL_WALLETS ? (
-      <ContainerBox alignItems="center" gap={1}>
-        <Typography variant="bodySmallBold" color={({ palette: { mode } }) => colors[mode].typography.typo2}>
-          {intl.formatMessage(
-            defineMessage({
-              defaultMessage: 'All',
-              description: 'allWallets',
-            })
+      <ContainerBox alignItems="center" gap={2}>
+        <ContainerBox alignItems="center" gap={1}>
+          <Typography variant="bodySmallBold" color={({ palette: { mode } }) => colors[mode].typography.typo2}>
+            {intl.formatMessage(
+              defineMessage({
+                defaultMessage: 'All',
+                description: 'allWallets',
+              })
+            )}
+          </Typography>
+          {allWalletsVerified ? (
```

Comment:
This is not just if all wallets are verified, but if you have a wallet selected and it is verified it should also apply the success icon